frontend/Makefile

```diff
@@ -16,7 +16,7 @@ frontend:
 	go build -o aro-hcp-frontend .
 
 run:
-	./aro-hcp-frontend --use-cache --region ${REGION} \
```

Comment:
the REGION variabel for personal DEV environments comes from dev-infrastructure/configuration/dev.mk, which is shared by basically all services (CS, maestro, hypershift-operator, ...)

if you want to change this then you need to change it throughout the entire repo

Reply:
I don't want the change to be that widespread, at least not in this PR.  I'll add this to the RP's Makefile:

```
# dev-infrastructure defines this as REGION
LOCATION ?= ${REGION}
```